Objects/obmalloc.c

```diff
@@ -1231,6 +1231,21 @@ _PyObject_XDecRefDelayed(PyObject *ptr)
 }
 #endif
 
+#ifdef Py_GIL_DISABLED
+void
+_PyObject_XSetRefDelayed(PyObject **ptr, PyObject *value)
```

Comment:
Can you move this function to Objects/object.c? This file is for memory allocators, it's not directly related.

Reply:
@vstinner I understand the suggestion was to move the definitions of `_PyObject_XSetRefDelayed` and `_PyObject_XDecRefDelayed` to another file. While I’d prefer to keep both in the same file, `_PyObject_XDecRefDelayed` references `free_delayed`, which is also referenced by `_PyMem_FreeDelayed`. Because of that, I’d rather keep the current structure.